### PR TITLE
add Delete modal by feedback

### PR DESCRIPTION
**Pull request by TaraniukD**

### CODE REVIEW

File: src/components/modals/FeedbackModal.jsx [hunk 1]
```diff
@@ -92,13 +112,22 @@ export const FeedbackModal = ({ visible, setVisible }) => {
...
             </FeedbackWrapper>
           )}
         </Modal>
+      )}
+
+            {(!isDeleteModalVisible && isFeedbackEditModalVisible) && (
+        <Modal
+          visible={visible}
+          onClose={() => setVisible(false)}
+          onEnterPress={() => setVisible(false)}
```
Comment (OlehZankiv) [inline src/components/modals/FeedbackModal.jsx]: What is the trash here?)

File: src/components/modals/DeleteModal.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import { useTranslation } from 'react-i18next'
+import { Text } from '../Text'
+import { useDeleteReview } from '../../hooks/query'
+
+
+export const DeleteModal = ({ visible, setVisible, setDeleteVisible, title, review }) => {
+    const { t } = useTranslation()
+    const { deleteReview } = useDeleteReview()
+
+    const handleDeleteReview = () => {
```
Comment (OlehZankiv) [inline src/components/modals/DeleteModal.jsx]: Modal should be dynamic, it shouldn't depends on element which you wanna delete

File: src/components/modals/DeleteModal.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    return (
+    <Modal visible={visible} onClose={() => setVisible(false)}>
+        <Text type='h5' style={{ textAlign: "center" }}>
+            {title}
+        </Text>
+            <ButtonsWrapper>
+              <Button
+                style={{ marginRight: 8, borderRadius: 8 }}
```
Comment (OlehZankiv) [inline src/components/modals/DeleteModal.jsx]: We have prettier please use  it

File: src/components/modals/DeleteModal.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                type='submit'
+                title={t('Delete')}
+                onClick={handleDeleteReview}
+              />
+              <Button
+                variant='secondary'
+                fullWidth
+                style={{ borderRadius: 8 }}
+                title={t('Cancel')}
+                onClick={() => {setDeleteVisible(false), setVisible(true)}}
```
Comment (OlehZankiv) [inline src/components/modals/DeleteModal.jsx]: Where did you find this syntax, it's totaly wrong)

File: src/components/modals/DeleteModal.jsx (hunk 3 above)
Comment (OlehZankiv) [inline src/components/modals/DeleteModal.jsx]: no need to set margin if you have display flex as parent component, use gaps

File: src/components/modals/DeleteModal.jsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    const { deleteReview } = useDeleteReview()
+
+    const handleDeleteReview = () => {
+        deleteReview(review.id)
+        setDeleteVisible(false)
+        setVisible(true)
+    }
+
+    return (
+    <Modal visible={visible} onClose={() => setVisible(false)}>
```
Comment (OlehZankiv) [inline src/components/modals/DeleteModal.jsx]: Add onEnter prop

File: src/components/modals/DeleteModal.jsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        <Text type='h5' style={{ textAlign: "center" }}>
```
Comment (OlehZankiv) [inline src/components/modals/DeleteModal.jsx:30]: Modal should have title and description
Title -> what the modal is
Description -> what will be if I will press the buttons

File: src/components/modals/DeleteModal.jsx [hunk 7]
```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components'
+import { Button, Modal } from '../index'
+import { useTranslation } from 'react-i18next'
+import { Text } from '../Text'
+import { useDeleteReview } from '../../hooks/query'
+
+
+export const DeleteModal = ({ visible, setVisible, setDeleteVisible, title, review }) => {
+    const { t } = useTranslation()
+    const { deleteReview } = useDeleteReview()
```
Comment (OlehZankiv) [inline src/components/modals/DeleteModal.jsx:10]: This need to remove, no need to assosiate this modal with something

File: src/components/modals/FeedbackModal.jsx [hunk 8]
```diff
@@ -19,18 +20,29 @@ export const FeedbackModal = ({ visible, setVisible }) => {
 
   const { reviews } = useReviews('owner')
   const { addReview, isLoading } = useAddReview(formik.current?.resetForm)
-  const { deleteReview } = useDeleteReview()
 
   const [isFeedbackEditModalVisible, setFeedbackEditModalVisible] = useState(false)
+  const [isDeleteModalVisible, setDeleteModalVisible] = useState(false)
   const [editedReview, setEditedReview] = useState('')
+  const [deletedReview, setDeleteReview] = useState('')
```
Comment (OlehZankiv) [inline src/components/modals/FeedbackModal.jsx:27]: editedReview and deketedReview it is not a strings

File: src/components/modals/FeedbackModal.jsx [hunk 9]
```diff
@@ -44,6 +56,14 @@ export const FeedbackModal = ({ visible, setVisible }) => {
         setVisible={setFeedbackEditModalVisible}
         review={editedReview}
       />
+      <DeleteModal
+        title={t('Confirm the deletion')}
```
Comment (OlehZankiv) [inline src/components/modals/FeedbackModal.jsx:60]: All text should be container into src/localization/strings/[en.js and uk.js], please add text to translations alphabetically

File: src/components/modals/FeedbackModal.jsx (hunk 8 above)
Comment (TaraniukD) [reply src/components/modals/FeedbackModal.jsx:27]: +

File: src/components/modals/FeedbackModal.jsx (hunk 9 above)
Comment (TaraniukD) [reply src/components/modals/FeedbackModal.jsx:60]: +

File: src/components/modals/FeedbackModal.jsx (hunk 8 above)
Comment (TaraniukD) [reply src/components/modals/FeedbackModal.jsx:27]: не мій код

File: src/components/modals/FeedbackModal.jsx (hunk 1 above)
Comment (TaraniukD) [reply src/components/modals/FeedbackModal.jsx:128]: це в мейні було, переписати?)

File: src/components/modals/DeleteModal.jsx (hunk 2 above)
Comment (TaraniukD) [reply src/components/modals/DeleteModal.jsx:12]: була така назва, вірно зроблю універсальною)

File: src/components/modals/DeleteModal.jsx (hunk 3 above)
Comment (TaraniukD) [reply src/components/modals/DeleteModal.jsx]: ок)

File: src/components/modals/DeleteModal.jsx (hunk 4 above)
Comment (TaraniukD) [reply src/components/modals/DeleteModal.jsx]: прийнято

File: src/components/modals/DeleteModal.jsx (hunk 3 above)
Comment (TaraniukD) [reply src/components/modals/DeleteModal.jsx]: це було з мейна), виправляю)

File: src/components/modals/DeleteModal.jsx (hunk 5 above)
Comment (TaraniukD) [reply src/components/modals/DeleteModal.jsx]: зроблено

File: src/components/modals/FeedbackModal.jsx (hunk 1 above)
Comment (TaraniukD) [reply src/components/modals/FeedbackModal.jsx]: ой, це тренувальне, не тут)

**Pull request closed without merging**